### DISCUSSION TITLE
Having no success setting up TigerVNC to access Xfce4 desktop

### DISCUSSION THREAD

**Discussion by sbabcoc** [TermuxArch/TermuxArch#93] [Q&A]
I want to run Arch Linux in the Xfce4 desktop environment.
* I ran the `csystemctl` command to install the `systemctl` replacement 
* I've installed the `xfce4` package
* I've installed the `tigervnc` package, setting the password, adding a user, and defining the configuration
* I start the VNC server with: `systemctl start vcnserver@:1.service`

Everything runs without complaint, but when I try to connect to the server, my VNC client reports "Connection refused"
What am I doing wrong?

UPDATE: I've now installed `lightdm` to see if this is any help.

I'm fuzzy on whether or not I need to specify all values in the configuration, like "geometry" in the TigerVNC config or "width", "height", and "depth" in the LightDM config.

**Comment by SDRausty** [comment]
Please see https://github.com/TermuxArch/TermuxArch/issues/28#issuecomment-691035121 and share your progress.

>Xvnc -localhost -geometry 1024x768 -depth 24 -SecurityTypes=None &

**Comment by SDRausty** [comment]
Here are a few more reprints that should be helpful:

It's just a simple command to setup the exports.  For it to work, it must be sourced.  That's why the dot `.` and space are important.   Otherwise, it won't work right:
```
[19:36 ~ ]$ . exd
[19:36 ~ ]$ export
declare -x DISPLAY=":0"
declare -x PULSE_SERVER="tcp:127.0.0.1:4712"
```

_Originally posted by @SDRausty in https://github.com/SDRausty/TermuxArch/issues/59#issuecomment-381285151_

 ## These issues are forked from this issue, and are labeled spoons:
1) [Android 8 System Information : `bash setupTermuxArch.sh sysinfo`](https://github.com/sdrausty/TermuxArch/issues/74#issue-315717462)
3) [The TermuxArch Keyring Command `keys` : Generating Keys](https://github.com/sdrausty/TermuxArch/issues/72#issue-315698618)
1) [Xorg : How to Run Xserver](https://github.com/sdrausty/TermuxArch/issues/73#issue-315708007)
2) [noVNC](https://github.com/sdrausty/TermuxArch/issues/71#issue-315694538)

Due to the size of this issue, comments are encouraged in spoon issues regarding these four topics.

_Originally posted by @SDRausty in https://github.com/SDRausty/TermuxArch/issues/59#issuecomment-382604999_

I would like to consolidate the steps used;  Please share your progress and the steps that you have used.

**Comment by SDRausty** [comment]
This https://github.com/SDRausty/termux-archlinux/issues/66 issue also has information about VNC support.
